### PR TITLE
Implement New Form and Test API Calls

### PR DESCRIPTION
**Pull request by bmsant**
This update includes the following enhancements:

- Form Implementation: Created a new form to capture user input and interact with the backend.

API Call Testing: Verified the API calls to ensure they work correctly with the new form.

**Changes**
Implemented a new form for user input.
Tested the API calls to confirm their functionality with the new form.

**Notes**
Please review the changes and provide feedback.
Ensure that all tests pass before merging.

### CODE REVIEW

File: src/lib/actions/captcha.actions.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import eokapi from "../../utils/eokapi";
+import { API_ENDPOINTS } from "@/constants";
+import { CaptchaResponse } from "../models/captcha.models";
+
+
```
Comment (naatsuo) [inline src/lib/actions/captcha.actions.ts:5]: ta com problema no enter ? ta saltando linha atoa ai

File: src/lib/actions/captcha.actions.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+export async function getCaptcha(): Promise<CaptchaResponse> {
+  try {
+    const response = await eokapi.get<CaptchaResponse>(API_ENDPOINTS.CAPTCHA);
+    const data = response.data;
+    sessionStorage.setItem('token', data.token);
```
Comment (naatsuo) [inline src/lib/actions/captcha.actions.ts:10]: valida o salto de linha

File: src/lib/actions/masterAccount.actions.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import eokapi from "../../utils/eokapi";
+import { API_ENDPOINTS } from "@/constants";
+import { MasterAccountRequests } from "../models/masterAccount.models";
+
+
```
Comment (naatsuo) [inline src/lib/actions/masterAccount.actions.ts:5]: ta com problema no enter ?

File: src/lib/actions/masterAccount.actions.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+export async function loginMasterAccount(data: MasterAccountRequests) {
+  try {
+    const response = await eokapi.post(API_ENDPOINTS.MASTER_ACCOUNTS.LOGIN, {
+      email: data.email,
+      password: data.password,
+      captchaReply: data.captchaReply,
+    });
+    const responseData = response.data;
+    sessionStorage.setItem('token', responseData.token);
```
Comment (naatsuo) [inline src/lib/actions/masterAccount.actions.ts:28]: aqui o problema do enter falhou

**Comment by naatsuo** [comment]
Alt Text

**Review by bmsant — COMMENTED** [review]
Resolvendo os mesmos B.O

**Review by naatsuo — APPROVED** [review]

**Review by naatsuo — APPROVED** [review]

**Pull request merged**